### PR TITLE
Add dot files

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+  - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+  - travis_install_nightly
+
+script:
+  - travis_run_flake8
```
Comment (pedrobaeza) [inline .travis.yml]: This is not needed.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[![Build Status](https://api.travis-ci.org/OCA/department.svg?branch=7.0)](https://travis-ci.org/OCA/department)
+[![Coverage Status](https://coveralls.io/repos/OCA/department/badge.png?branch=7.0)](https://coveralls.io/r/OCA/department?branch=7.0)
+
+Odoo Department Modules
```
Comment (pedrobaeza) [inline README.md]: You can put `OCA department modules for OpenERP` to give OCA a common brand, like in https://github.com/OCA/account-invoicing/tree/7.0

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+=======================
+
+This project aims to add Department context information to Odoo Models.
+
+This can help organizations where processes are strongly organized by Department.
```
Comment (pedrobaeza) [inline README.md:9]: Continuing with the branding, you can put:

```
----

OCA, or the Odoo Community Association, is a nonprofit organization whose 
mission is to support the collaborative development of Odoo features and 
promote its widespread use.

http://odoo-community.org/
```

File: README.md (hunk 2 above)
Comment (pedrobaeza) [inline README.md]: I say OpenERP, because this is 7.0 version, that it's still named as OpenERP.

**Comment by dreispt** [comment]
Build fails because of flake8. I guess a cleanup is needed.

**Comment by dreispt** [comment]
For bonus points, clean up was made on current code.

**Comment by yvaucher** [comment]
still @pedrobaeza's comments to resolve
- travis_run_flake8 to remove
- unit tests ?

**Comment by bwrsandman** [comment]
@yvaucher 

> - unit tests ?

They are added

File: .travis.yml (hunk 1 above)
Comment (yvaucher) [inline .travis.yml]: I still see this line in the diff

**Comment by dreispt** [comment]
Removed travis_run_flake8

**Comment by bwrsandman** [comment]
:+1:

**Comment by pedrobaeza** [comment]
I merge the branch, and the fix will come in another PR.

**Pull request merged**